### DISCUSSION TITLE
Windows 11 compatability

### DISCUSSION THREAD

**Discussion by Trance-Paradox** [microsoft/winget-cli#1229] [Q&A] [answered]
Just a simple question will winget be compatible with windows 11 releasing later this year?

**Comment by denelon** [comment, marked as answer]
Yes - A simple answer 😊.

**Reply by ghost** [reply]
@denelon, could you please explain how the new Microsoft Store policy on Windows 11 (allowing publishing Win32, Android, PWA, etc.) will coexist with winget?

To be honest, I do not quite understand the purpose on the project in this case, 'cause the packages will be stored in two places: Microsoft internal servers for the Microsoft Store and in the GitHub repo for winget. How will synchronization be performed?

In the past, with old Microsoft Store policy, the Store was mostly useless due to lack of software compared with winget.

**Reply by Ne4to** [reply]
Will it be pre installed?

**Comment by denelon** [comment]
@altwarg think of the Microsoft Store and Windows Package Manager as two different user experiences. One is a GUI, and the other is command line. They both have slightly different target audiences. 

The Windows Package Manager has the concept of a source. A source represents a collection of packages available for Windows Package Manager users. One of the sources for the Windows Package Manager is the Microsoft Community Repository (named "winget"), and another is the Microsoft Store (see note below). The Windows Package Manager has the ability to span multiple sources. In the event of a duplicate package, the user will be asked to disambiguate and specify the source. We will also add settings and default behaviors to simplify this process.

The Microsoft Community Repository will continue to be open for community submission to the default source for the Windows Package Manager. The Microsoft Store has a different user experience through Partner Center where publishers and Independent Software Vendors submit packages. 

>Note: At this time, the original experimental source is a PreIndexed Package named "msstore". The source for the Microsoft Store preview is a REST API based source named "storepreview".

**Reply by just1a-person** [reply]
for me it's "newstore"

**Reply by denelon** [reply]
@itsme-alan, we're making some changes to help here. There are currently two different sources representing the store. One is available via experimental settings called "msstore" (a preIndexed package). The one that you are referencing ("newstore" is a REST based source) is installed via the Windows 11 version of the "App Installer". It will likely be changed to "storepreview" to help clarify. Once we have consolidated support into the single REST source, the other will be removed/renamed.

**Comment by denelon** [comment]
@Ne4to it will be an automatic update included in the App Installer from the Microsoft Store.

**Reply by Trance-Paradox** [reply]
Means only those who have subscribed to it will get it or it will be distributed to everyone else

**Reply by denelon** [reply]
It will be distributed to everyone on Windows 10 1809 and above (including all Windows 11 versions). If you are running a Windows 11 insider build it's already in the default update for App Installer.
